### ISSUE TITLE
Feedback for ifs makes reference to loops

### ISSUE THREAD

**Issue by sergutsan** [SophieKoonin/JavaDecaf#13]
Feedback for ifs makes reference to loops, which is a bit confusing. 

What do you see?

```
Error: Encountered ")" followed by "return" at line 4, column 14. 
Loop statements must be contained in curly braces, e.g. :
 if (x > y) {
    println(x);
}
```

What should you see instead?

```
Error: Encountered ")" followed by "return" at line 4, column 14. 
If-branch statements must be contained in curly braces, e.g. :
 if (x > y) {
    println(x);
}
```

If should say "if branch" instead of "loop".

**Comment by sophiekoonin** [comment]
That's easily changed in the blob class that is ParseException.java - just requires a different wording.

**Comment by sergutsan** [comment]
Thanks for the pointer. 

Fixed in 14b195be7b6.
